client-common/src/main/java/org/apache/cassandra/sidecar/common/request/data/CreateSliceRequestPayload.java

```diff
@@ -50,8 +50,8 @@ public class CreateSliceRequestPayload
     private final String bucket;
     private final String key;
     private final String checksum;
-    private final BigInteger startToken;
-    private final BigInteger endToken;
+    private final BigInteger firstToken; // inclusive
```

Comment:
On Analytics side, the client create slices with the first token and last token. Both are inclusive.